nntrainer/src/pooling2d_layer.cpp

```diff
@@ -177,9 +230,9 @@ Tensor Pooling2DLayer::pooling2d(Tensor in, int &status) {
           for (unsigned int pi = 0; pi < p_height; ++pi) {
             for (unsigned int pj = 0; pj < p_width; ++pj) {
               sum += in.getValue(0, i, j + pi, k + pj);
-	    }
-	  }
-	  sum = sum / (p_height + p_width);
+            }
+          }
+          sum = sum / (p_height + p_width);
```

Comment:
may be `sum = sum / (p_height * p_width)` assuming that average is over all the elements included in the sum.

Reply:
also lets cache `p_height * p_width` as `p_size` as backwarding.

Reply:
right. it should be multiplication. Thanks.